ui/src/app/machines/views/MachineDetails/MachineSummary/TestResults/TestResults.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { Button, Icon, ICONS, Tooltip } from "@canonical/react-components";
+
+import type { SetSelectedAction } from "..";
+import { capitaliseFirst } from "app/utils";
+import { useSendAnalytics } from "app/base/hooks";
+import type { MachineDetails } from "app/store/machine/types";
+import { HardwareType } from "app/base/enum";
+
+type Props = {
+  machine: MachineDetails;
+  hardwareType: HardwareType;
+  setSelectedAction: SetSelectedAction;
+};
+
+const getScriptType = (hardwareType: HardwareType) =>
+  HardwareType[hardwareType].toLowerCase();
```

Comment:
Just thought you should know I spent an embarrassing amount of time trying to parse this thinking `HardwareType` was a normal TS type. :grimacing: